src/app/docs/popover/popover-docs.component.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  ChangeDetectionStrategy,
+  Component
+} from '@angular/core';
+
+import {
+  SkyDocsDemoControlPanelChange
+} from '@skyux/docs-tools';
+
+@Component({
+  selector: 'app-popover-docs',
```

Comment:
The `sky-` prefix is a requirement for libraries since it adds a visual namespace that won't confuse our consumers. For SPA components, I'd prefer to use something different so that content authors don't think these components are part of the `@skyux/docs-tools` or `@skyux/popovers` libraries.